touchview/src/main/java/com/ortiz/touchview/TouchImageView.java

```diff
@@ -1,1463 +1,1511 @@
-package com.ortiz.touchview;
-
-import android.annotation.TargetApi;
-import android.content.Context;

```

Comment:
What happens ? You changed **every** line, a valid diff is impossible !
Maybe wrong line separator ?

Please double check it 
![image](https://user-images.githubusercontent.com/3314607/64560551-ff8c4680-d348-11e9-996f-94b7156d6ee0.png)